README.md

```diff
@@ -478,6 +478,29 @@ $ gunk convert /path/to/file.proto
 $ gunk convert /path/to/protobuf/directory
 ```
 
+If your `.proto` is referencing another `.proto` from another directory, 
+you can add `import_path` in the global section of your `.gunkconfig`.
+If you don't provide `import_path` it will only search in the root directory.
+
+
+```ini
+import_path=relative/path/to/protobuf/directory
```

Comment:
clarify that this is relative to the gunk project root, i.e. where the config is